lib/src/realm_class.dart

```diff
@@ -33,6 +33,7 @@ import 'results.dart';
 export 'package:realm_common/realm_common.dart'
     show Ignored, Indexed, MapTo, PrimaryKey, RealmError, RealmModel, RealmUnsupportedSetError, RealmStateError, RealmCollectionType, RealmPropertyType;
 export "configuration.dart" show Configuration, RealmSchema, SchemaObject;
+export 'entity.dart' show RealmEntity;
```

Comment:
Could we place this class in `realm_object.dart` and remove `entity.dart`. Seems we have a chance here to keep the files number low.

Reply:
If we import `realm_object.dart` from `list.dart` then yes, but since it is just a file in `lib/src` (ie. not `lib`) I'm not sure I see the value in keeping the file count low? Anyway, I have prepped a commit, where `RealmEntity` is moved to `realm_object.dart`.